astropy/nddata/mixins/ndarithmetic.py

```diff
@@ -8,14 +8,138 @@
+                doc = func.__doc__
+                func.__doc__ = None
+            if not doc:
+                raise ValueError
+            kwargs['original_doc'] = func.__doc__ or ''
+            func.__doc__ = doc.format(*args, **kwargs)
+            return func
+        return set_docstring
+
+# TODO: Maybe not nice to pollute globals but I felt the same way about
+# polluting the class namespace and this way potential subclasses may or refuse
+# to pull this docstring into their class as well.
+
+# Docstring templates for add, subtract, multiply, divide methods.
+_arit_doc = """
+    Performs {name} by evaluating ``self`` {op} ``operand``.
+
+    For the inverse operation or operations between different subclasses take a
+    look at
+    :meth:`~astropy.nddata.NDArithmeticMixin.ic_{name}` .
+
+    Parameters
+    ----------
+    operand : `NDData`-like instance or convertable to one.
+        The second operand in the operation.
+
+    propagate_uncertainties : `bool` or ``None``, optional
+        If ``None`` the result will have no uncertainty. If ``False`` the
+        result will have a copied version of the first operand (or the
+        second if the first had no uncertainty). In case this is ``True``
+        the result will have a correctly propagated uncertainty from the
+        uncertainties of the operands but this assumes that the uncertainties
+        are `StdDevUncertainty`. Default is ``True``.
+
+    handle_mask : callable, ``False`` or ``None``, optional
+        If ``None`` the result will have no mask. If ``False`` the result
+        will have a copied version of the mask of the first operand (or
+        if the first operand has no mask then the one from the second is
+        taken). If it is a callable then the specified callable must
+        create the results ``mask`` and if necessary provide a copy.
+        Default is `numpy.logical_or`.
+
+    handle_meta : callable, ``False`` or ``None``, optional
+        If ``None`` the result will have no meta. If ``False`` the result
+        will have a copied version of the meta of the first operand (or
+        if the first operand has no meta then the one from the second is
+        taken). If it is a callable then the specified callable must
+        create the results ``meta`` and if necessary provide a copy.
+        Default is ``False``.
+
+    compare_wcs : callable, ``False`` or ``None``, optional
+        If ``None`` the result will have no wcs and no comparison between
+        the wcs of the operands is made. If ``False`` the result will have
+        the wcs of the first operand (or second if the first had ``None``).
+        If it is a callable then the specified callable must
+        compare the ``wcs``. The resulting ``wcs`` will be like if ``False``
+        was given otherwise it raises a ``ValueError`` if the comparison was
+        not successful. Default is ``False``.
+
+    uncertainty_correlation : ``Number`` or `~numpy.ndarray`, optional
```

Comment:
Now that you mention it: Should I put the formula inside a ``.. math::``? And I just realized that I used the ``Covariance`` (dAB = dA \* dB \* cor) instead of the ``Correlation`` (cor = dAB / (dA * dB)) formula?

What about:
"Correlation between the uncertainties:
.. math:: 
    cor = d_{AB} / (d_A * d_B)"